public/index.html

```diff
@@ -13,13 +13,16 @@
       window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
       ga('create', "%REACT_APP_GOOGLE_ANALYTICS_ID%", 'auto');
       ga('send', 'pageview');
+<<<<<<< HEAD
+=======
 
 
       ga(function(tracker) {
         // Remove after testing in stage
         console.log(tracker);
       });
 
+>>>>>>> develop
```

Comment:
@Adamj1232 merge conflict residue here 😃

Reply:
Ahh new computer problems! lol I setup autosave in my IDE so I dont miss these changes!